query-graphs-js/src/pathContext.ts

```diff
@@ -8,12 +8,24 @@ export function isPathContext(v: any): v is PathContext {
   return v instanceof PathContext;
 }
 
-function addExtractedConditional(operation: OperationElement, directiveName: string, addTo: [string, any][]) {
-  const applied = operation.appliedDirectivesOf(directiveName);
+export type OperationConditional = {
+  kind: 'include' | 'skip',
+  value: any,
```

Comment:
`'true'`, `'false'`, `'$foo'` - are you expecting something other than a string here? (should t/f be booleans?)
```suggestion
  value: string,
```

```suggestion
  value: string | boolean,
```

Reply:
It's actually properly "parsed" values at this point, so the proper type here is `Variable | boolean`, but you're right, we can do better than `any`. Updated.